scene_select/check_ancillary.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+
+import datetime
+from pathlib import Path
+import structlog
+import h5py
+import numpy
+import pandas
+import pytz
+
+
+LOG = structlog.get_logger()
+DEFINITIVE_START_DATE = datetime.datetime(2002, 7, 1)
```

Comment:
`DEFINITIVE_START_DATE` in this module's context becomes a little ambiguous.  Maybe `BRDF_DEFINITIVE_START_DATE`?

Reply:
Good idea, done